src/components/orgunits/OrgUnitData.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@dhis2/d2-i18n';
+import { CircularLoader } from '@dhis2/ui';
+import PeriodSelect from '../periods/PeriodSelect';
+import { apiFetch } from '../../util/api';
+import styles from './styles/OrgUnitData.module.css';
+
+/*
+ *  Displays a period selector and org unit data items (data elements, indicators, reporting rates, program indicators)
+ */
+const OrgUnitData = ({ id, periodType, defaultPeriod, data }) => {
+    const [period, setPeriod] = useState(defaultPeriod);
+    const [items, setItems] = useState(data);
+    const [isLoading, setIsLoading] = useState(false);
+
+    // Load data items if period is changed
+    useEffect(() => {
+        if (period.id === defaultPeriod.id) {
+            setItems(data);
+        } else {
+            setIsLoading(true);
+            apiFetch(`/organisationUnitProfile/${id}/data?period=${period.id}`)
+                .then(data => {
+                    setItems(data.dataItems);
```

Comment:
Maybe slightly confusing to have `data` var name reused? (see line 30 and line 24)

Reply:
Fixed!